website/static/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="refresh" content="0; url=docs/cli.html">
+    <script type="text/javascript">
+      window.location.href = 'docs/cli.html';
+    </script>
+    <title>Your Site Title Here</title>
+  </head>
+  <body>
+    If you are not redirected automatically, follow this <a href="docs/cli.html">link</a>.
+  </body>
+</html>
```

Comment:
Need another approach to handle the landing page
https://docusaurus.io/docs/en/site-creation#docs-landing-page